dotnet-desktop-guide/winforms/migration/snippets/clipboard-dataobject-net10/net/csharp/ITypedDataObjectExamples.cs

```diff
@@ -0,0 +1,74 @@
+using System;
+using System.Reflection.Metadata;
+using System.Windows.Forms;
+
+namespace ClipboardExamples
+{
+    // <ITypedDataObjectImplementation>
+    public class TypedDataObject : DataObject, ITypedDataObject
```

Comment:
`DataObject` implements `ITypedDataObject` so this isn't a good example. I don't think writing samples for `ITypedDataObject` is worth pre-emptively doing.